ci/gpu/build.sh

```diff
@@ -118,20 +118,24 @@ else
     # list test directory
     ls tests/
 
-    # Test with IB
-    # UCX_MEMTYPE_CACHE=n UCX_TLS=rc,cuda_copy,cuda_ipc py.test --cache-clear --junitxml=${WORKSPACE}/junit-ucx-py.xml -v --cov-config=.coveragerc --cov=ucp --cov-report=xml:${WORKSPACE}/ucp-coverage.xml --cov-report term tests/
+    # Setting UCX options
+    export UCXPY_IFNAME=eth0
+    export UCX_MEMTYPE_CACHE=n
+    export UCX_TLS=tcp,cuda_copy,sockcm
+    export UCX_SOCKADDR_TLS_PRIORITY=sockcm
 
     # Test with TCP/Sockets
     logger "TEST WITH TCP ONLY..."
-    UCXPY_IFNAME=eth0 UCX_MEMTYPE_CACHE=n UCX_TLS=tcp,cuda_copy,sockcm UCX_SOCKADDR_TLS_PRIORITY=sockcm py.test --cache-clear --junitxml=${WORKSPACE}/junit-ucx-py.xml -v --cov-config=.coveragerc --cov=cudf --cov-report=xml:${WORKSPACE}/ucp-coverage.xml --cov-report term tests/
+    py.test --cache-clear --junitxml=${WORKSPACE}/junit-ucx-py.xml -v --cov-config=.coveragerc --cov=ucxpy --cov-report=xml:${WORKSPACE}/ucp-coverage.xml --cov-report term tests/
 
     # Test downstream packages, which requires Python v3.7
     if [ $(python -c "import sys; print(sys.version_info[1])") -ge "7" ]; then
         logger "TEST OF DASK/UCX..."
-            UCXPY_IFNAME=eth0 UCX_MEMTYPE_CACHE=n UCX_TLS=tcp,cuda_copy,sockcm UCX_SOCKADDR_TLS_PRIORITY=sockcm py.test --cache-clear --junitxml=${WORKSPACE}/junit-ucx-py.xml -v --cov-config=.coveragerc --cov=cudf --cov-report=xml:${WORKSPACE}/ucp-coverage.xml --cov-report term  `python -c "import distributed.comm.tests.test_ucx as m;print(m.__file__)"`
+        py.test --cache-clear --junitxml=${WORKSPACE}/junit-ucx-py.xml -v --cov-config=.coveragerc --cov=ucxpy --cov-report=xml:${WORKSPACE}/ucp-coverage.xml --cov-report term \
```

Comment:
Here as well. Also if the xml names are the same they will be overwritten. They need to have different names.